theano/tensor/tests/test_sort.py

```diff
@@ -330,7 +330,7 @@ def test_argtopk_1d(self, size, k, dtype, sorted, idx_dtype):
         goal = np.argsort(xval)[idx].astype(idx_dtype)
 
         # due to uniqueness, we expect indices same
-        assert np.all(xval[np.sort(yval)] == xval[np.sort(goal)])
+        utt.assert_allclose(xval[np.sort(goal)], xval[np.sort(yval)])
```

Comment:
I don't think this is a good change due to the comment. Do you understand why this change fix the test?

Reply:
No, I don't know why it fixes it.

And I just realized that I cannot reproduce the error because `xval` is randomly generated without a fixed seed.